index.html

```diff
@@ -45,21 +45,22 @@ <h1 > hey there! <br/>
 	<!-- about me section -->
 	<div id="about"></div>
 
-		<div class="container_main">
-			<div class="about_me_box">
-				<div class="about_me">
-					<h3>About me</h3>
-					<p> Hi, I&#39;m Hannah! After many years working as a Graphic Designer,
-						I decided to make a career change (an idea both exciting and slightly terrifying!).
-						I&#39;m currently training to be a full stack developer at Mayden Academy in Bath, a 16 week intensive course to learn the skills required for the industry.
-					</p>
-					<p>Please take a look at my work completed so far... </p>
-				</div>
-				<div class="img_me_box">
-					<div class="img_me"></div>
-				</div>
+	<div class="container_main">
+		<div class="about_me_box">
+			<div class="about_me">
+				<h3>About me</h3>
```

Comment:
h3 used instead of h2. I like to order mine but Ashley said its up for debate. As long as h1 is used once (for SEO) then the rest doesn't matter too much.